test/utils.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons'
+
+var TestUtils = React.addons.TestUtils;
+
+export default {
+
+	getComponent: function (Component, props) {
```

Comment:
I'm not convinced these helpers are needed. The barely do any useful work, just adding extra conventions.